cyclops-ui/src/components/pages/ModuleDetails.tsx

```diff
@@ -880,6 +908,14 @@ const ModuleDetails = () => {
         cancelButtonProps={{ style: { display: "none" } }}
         width={"40%"}
       >
+        <div style={{ marginBottom: '10px' }}>
+        <Checkbox
+        onChange={handleCheckboxChange} checked={showManagedFields} >
+          Include Managed Fields
+        </Checkbox>
+      </div>
```

Comment:
Could you add an [Divider](https://ant.design/components/divider) under this div?